package.json

```diff
@@ -235,17 +241,20 @@
     "start-mysql": "bash dev/mysql/5.7/start.sh",
     "start-mysql-8": "bash dev/mysql/8.0/start.sh",
     "start-postgres": "bash dev/postgres/10/start.sh",
+    "start-yugabyte": "bash dev/yugabyte/start.sh",
```

Comment:
Afaik, we should use yugabytedb for referring database.
Yugabyte - company name
YugabyteDB - product name
 
Same for other places.